internal/export/database/export.go

```diff
@@ -253,10 +253,10 @@ func (db *ExportDB) ListAllSigntureInfos(ctx context.Context) ([]*model.Signatur
 
 	rows, err := conn.Query(ctx, `
     SELECT
-      id, signing_key, app_package_name, bundle_id, signing_key_version, signing_key_id, thru_timestamp
+      id, signing_key, LOWER(app_package_name), LOWER(bundle_id), signing_key_version, signing_key_id, thru_timestamp
```

Comment:
This could/might affect export generation. Any ideas?

Reply:
@gurayAlsac - any insight on if this impacts export importing?

Reply:
All lower is what we were expecting anyway in the EN API public key configuration lookups, and we don't look at app bundle id at all.